src/test/resources/cassandra-reaper-cassandra-at.yaml

```diff
@@ -8,6 +8,7 @@ repairRunThreadCount: 15
 hangingRepairTimeoutMins: 1
 storageType: cassandra
 incrementalRepair: false
+allowUnreachableNodes: true
```

Comment:
a) Previously the integration tests were working for me even though two of the three nodes could not be reached via JMX.

b) With this patch the integration tests then started to fail.

c) The integration tests with this patch do work with `allowUnreachableNodes=true` ,  as one would expect.

I don't know, but I'm left wondering if the problem was in (a), and not actually in this patch. It's worth checking though.